frontend/src/use-cases/home/views/Prompt/prompt.view.js

```diff
@@ -47,15 +47,16 @@ const Prompt = () => {
     };
 
     return (
-        <div className="prompt">
-            <div className="innerPrompt">
+        <PromptWrapper size={{ height: "33rem" }}>
+            <DigitDesign.CardBody>
```

Comment:
Why don't use a `DigitEditDataCard`?

Reply:
This prompt existed before DigitEditDataCard. Changing to DigitEditDataCard is a lot of work for nothing. This can be posted in an issue